python/GafferUI/NodeGraph.py

```diff
@@ -52,11 +52,16 @@ def __init__( self, scriptNode, **kw ) :
 			] ),
 		)
 
+		layoutNodes =  not self.__nodesHaveUIPosition( scriptNode )
```

Comment:
Although... If I remove this, then it's possible that some nodes will end up unplaced. My code would only auto-layout the nodes if *all* nodes have no position. As soon as one node has a position, it doesn't do the auto-layout. I'm feeling that, in the case where some nodes have positions and some don't, I should either auto-layout the ones that don't or just set them all to 0,0.